### ISSUE TITLE
"Page break" or "pagebreak"?

### ISSUE THREAD

**Issue by plannero** [w3c/wcag#1683]
The published WCAG 2.2 mentions and has a definition of "pagebreak locators" at 
https://www.w3.org/TR/WCAG22/#dfn-pagebreak-locator 

The corresponding Understanding document instead mentions "page break locators" (note the spacing) and defines that term quite differently at
https://www.w3.org/WAI/WCAG22/Understanding/fixed-reference-points.html#dfn-page-break-locators 

Furthermore, the definition in the Understanding document (which appears also in the GitHub Editor's Draft) contains a typo: 
Too many "is" in this: "A digital book is published is with no print equivalent "

**Comment by alastc** [comment]
Hi @plannero,

We have standardised on "page break", and the raw documents are up to date (there is no mention of "pagebreak"), but it won't appear in the /TR/ version until we do another update, which should be soon.

In the meantime you can see the latest version here:
https://w3c.github.io/wcag/understanding/fixed-reference-points.html 

-Alastair

**Comment by plannero** [comment]
Very good. Only the little typo remains to be fixed then: 
"A digital book **is** published **is** with no print equivalent"
In my eyes there are too many "is" in that sentence. (But perhaps I read it wrongly? English is not my native language.)

**Comment by alastc** [comment]
Ach, I've fixed that one twice now! Must have had a merge go wrong. Ok, fixed in 'main' now (again?), thanks.
